submissions/description/application_level_denial_of_service_dos/critical_impact_and_or_easy_difficulty/guidance.json

```diff
@@ -0,0 +1,6 @@
+{
+  "overview": "Provide a 1-2 sentence description - see [our key details phrasing guide](http://cveproject.github.io/docs/content/key-details-phrasing.pdf) for tips. This format is a good guide: [VULNTYPE] in [COMPONENT] in [APPLICATION] allows [ATTACKER] to [IMPACT] via [VECTOR]",
+  "walkthrough_poc": "Provide a step-by-step walkthrough on how to access the vulnerable injection point, and how to exploit the vulnerability. Adding a dot-pointed walkthrough with relevant screenshots will speed triage time and result in faster rewards!",
```

Comment:
Note the simplified key. "Walkthrough & PoC" gets remapped to "walkthrough_poc".